### PR TITLE
Level-wise view added for Admin or Applicants PR1

### PR DESCRIPTION
**Pull request by amitsaini4556**
### View For Admin
Screenshot (15)

### View For Applicants
Screenshot (16)

fixes issue #14

### CODE REVIEW

File: facultyRecruitment/userAccounts/views.py [hunk 1]
```diff
@@ -34,7 +39,8 @@ def signUp(request):
 		if form.is_valid():
 			user = form.save()
 			username = form.cleaned_data.get('username')
-
+			group = Group.objects.get(name = 'candidates')
+			user.groups.add(group)
```
Comment (MuskanPaliwal) [inline facultyRecruitment/userAccounts/views.py:43]: would like to know that why have we created a group of users? We only have two types of user role in the application. How using groups here is a better idea here?

Comment (amitsaini4556) [reply facultyRecruitment/userAccounts/views.py:43]: There are two groups one is`admin` and another `candidates` already created by django backend admin's site.
All users added in `admin` behave like a staff and on the other hand `candidates` group as a active user only.

**Review by hemabhagnani — APPROVED** [review]

File: facultyRecruitment/userAccounts/views.py (hunk 1 above)
Comment (MuskanPaliwal) [reply facultyRecruitment/userAccounts/views.py:43]: Okay. Get it.

File: facultyRecruitment/templates/jobs/addJobForm.html [hunk 2]
```diff
@@ -9,26 +9,27 @@
 					<span class="login100-form-title p-b-32">
 						Post a Job
 					</span>
+
```
Comment (MuskanPaliwal) [inline facultyRecruitment/templates/jobs/addJobForm.html:12]: Unwanted blank line. Please remove it

File: facultyRecruitment/templates/jobs/addJobForm.html [hunk 3]
```diff
@@ -9,26 +9,27 @@
...
 					<span class="txt1 p-b-11">
 						Position*
 					</span>
 					<div class="wrap-input100 validate-input m-b-24" data-validate = "Position is required">
 						<input class="input100" style="height: 45px;"  type="text" name="position" >
-
 					</div>
+
```
Comment (MuskanPaliwal) [inline facultyRecruitment/templates/jobs/addJobForm.html:19]: please remove the blank line.

File: facultyRecruitment/templates/jobs/addJobForm.html [hunk 4]
```diff
@@ -122,4 +121,16 @@
...
+  var month = dtToday.getMonth() + 1;
+  var day = dtToday.getDate();
+  var year = dtToday.getFullYear();
+  if(month < 10)
+      month = '0' + month.toString();
+  if(day < 10)
+      day = '0' + day.toString();
+  var minDate= year + '-' + month + '-' + day;
+  document.getElementById('applyBy').setAttribute('min', minDate);
+</script>
```
Comment (MuskanPaliwal) [inline facultyRecruitment/templates/jobs/addJobForm.html:135]: please indent it properly.

File: facultyRecruitment/templates/jobs/addJobForm.html [hunk 5]
```diff
@@ -105,14 +104,14 @@
 							</button>
             </div>
 						<div class="container-login100-form-btn" style="justify-content: center;">
-								<button type="reset" value="Login" class="login100-custom-form-btn">
-										Reset
-								</button>
+							<button type="reset" value="Login" class="login100-custom-form-btn">
+									Reset
+							</button>
 						</div>
```
Comment (MuskanPaliwal) [inline facultyRecruitment/templates/jobs/addJobForm.html]: only one Tab is required for `Reset` word.

**Review by MuskanPaliwal — CHANGES_REQUESTED** [review]
Please check the indentation once.

File: facultyRecruitment/templates/jobs/addJobForm.html (hunk 3 above)
Comment (amitsaini4556) [reply facultyRecruitment/templates/jobs/addJobForm.html:19]: the blank line used for clear understanding.

File: facultyRecruitment/templates/jobs/addJobForm.html (hunk 5 above)
Comment (amitsaini4556) [reply facultyRecruitment/templates/jobs/addJobForm.html]: ### It's looking good in my ide.
image

File: facultyRecruitment/templates/jobs/addJobForm.html (hunk 4 above)
Comment (amitsaini4556) [reply facultyRecruitment/templates/jobs/addJobForm.html:135]: It will be done in nxt PR.

**Review by MuskanPaliwal — APPROVED** [review]

**Pull request merged**